schedule/yast/snapper_cleanup_timeline.yaml

```diff
@@ -0,0 +1,18 @@
+name:       snapper_cleanup_timeline
+description:   >
+  Test performs checks for the functionality of snapper cleanup timeline algorithm.
+schedule:
+  - boot/boot_to_desktop
+  - console/snapper_cleanup_timeline
+test_data:
+  snapper_config:
```

Comment:
this node doesn't seem necessary now, at least you have in mind some other section for the future.

Reply:
Compared to other test_data formats, it would seem quite messy if I remove it.